linodecli/configuration/config.py

```diff
@@ -290,7 +290,10 @@ def update(
         ):
             print(f"User {username} is not configured.")
             sys.exit(ExitCodes.USERNAME_ERROR)
-        if not self.config.has_section(username) or allowed_defaults is None:
+        if (
```

Comment:
Not sure how it used to work, but currently the `DEFAULT` section can't be found by calling `has_section("DEFAULT")`. It will skip update and use the defaults. Add a check to look for the default section under which we put the configured values at the most time.

Luckily we can still use `config.has_option("DEFAULT", key)` to get the value